Sources/Subs.php

```diff
@@ -4852,9 +4852,6 @@ function inet_ptod($ip_address)
  */
 function inet_dtop($bin)
 {
-	if(strpos($bin,'.')!==false || strpos($bin,':')!==false)
-		return $bin;
```

Comment:
to remove this is a error,
in the furture unknown didn't exists anymore see https://github.com/SimpleMachines/SMF2.1/pull/3373
and this lines are needed for pg